cli/src/processor/thread.rs

```diff
@@ -101,6 +101,16 @@ pub fn update(
         settings,
     );
     client.send_and_confirm(&[ix], &[client.payer()]).unwrap();
-    get(client, id)?;
+    get(client, thread_pubkey)?;
     Ok(())
 }
+
+pub fn parse_pubkey_from_id_or_address(
```

Comment:
Haven't found an ideal place to put this function.

- Putting this In `programs/thread/src/objects/thread.rs` seems a bit weird, since it's just for parsing.
- Putting it in `processor/processor.rs` seems similar to putting it in `processor/thread.rs`  idk.